### ISSUE TITLE
Popup "Compte inactif"

### ISSUE THREAD

**Issue by Insoleet** [duniter/cesium#574]
Salut,

Hier, lors de la présentation de la Ğ1 à de nouveaux arrivés, je leur ai fait créé un compte "Simple portefeuille", et je leur ai versé 15 Ğ1 dessus.

A force de rafraichir, les 15 Ğ1 sont apparues "En attente de confirmation" -> tout va bien.

Plus tard, ils ont voulu se reconnecter pour pouvoir faire une transaction, et la popup "Compte inactif" est apparue. Ce qui m'a beaucoup surpris puisque ce compte n'était de toute évidence pas inactif, une transaction était en attente dessus !

Ca me semble être un bug !

**Comment by c-geek** [comment]
Tu es sûr que c'était cela ? J'ai souvenir qu'il y avait aussi une histoire d'identifiants déjà pris par quelqu'un d'autre. :confused:

**Comment by blavenie** [comment]
Oui, c'ets un bug. La définition d'une "compte inactif" ne regardes que les sources.
IL faut que je regarde aussi les TX en attentes.
De même, si un profil Cesium+ est défini, cesium ne devrais pas avertir
